.vscode/settings.json

```diff
@@ -1,7 +1,20 @@
 {
-    "python.testing.pytestArgs": [
-        "."
+    "python.linting.pylintEnabled": true,
+    "python.linting.flake8Enabled": true,
+    "python.linting.mypyEnabled": true,
+    "python.linting.enabled": true,
+    "python.formatting.provider": "black",
+    "editor.formatOnSave": true,
+    "editor.rulers": [
+        88
     ],
     "python.testing.unittestEnabled": false,
-    "python.testing.pytestEnabled": true
```

Comment:
This should be here, otherwise VSCode testing does not know what to use for tests.

Reply:
It still is set to ``true``! See below

Reply:
omg